tests/cross_functional/system_test/test_graceful_nodes_shutdown.py

```diff
@@ -0,0 +1,439 @@
+"""
+Test to verify that data is accessible and uncorrupted as well as
+operational cluster after graceful nodes shutdown
+"""
+
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs.ocp import OCP, wait_for_cluster_connectivity
+from ocs_ci.ocs import constants, defaults, registry
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    ResourceWrongStatusException,
+    UnexpectedBehaviour,
+)
+from ocs_ci.ocs.longevity import start_ocp_workload
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.monitoring import (
+    validate_pvc_created_and_bound_on_monitoring_pods,
+    validate_pvc_are_mounted_on_monitoring_pods,
+)
+from ocs_ci.utility.uninstall_openshift_logging import uninstall_cluster_logging
+from ocs_ci.utility.retry import retry
+from ocs_ci.framework.pytest_customization.marks import (
+    system_test,
+    polarion_id,
+    skipif_no_kms,
+    skipif_ocs_version,
+    magenta_squad,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs.node import (
+    get_nodes,
+)
+from ocs_ci.ocs.resources.fips import check_fips_enabled
+
+logger = logging.getLogger(__name__)
+
+
+class TestGracefulNodesShutdown(E2ETest):
+    """
+    Test uncorrupted data and operational cluster after graceful nodes shutdown
+    """
+
+    bucket_names_list = []
+
+    @pytest.fixture(autouse=True)
+    def checks(self):
+        try:
+            check_fips_enabled()
+        except Exception as e:
```

Comment:
This exception is broad. can you use a specific exception that check_fips_enabled() throws